### PR TITLE
Update maintenane presets for devices from directories: Stairville, S…

### PR DESCRIPTION
**Pull request by kluszon**
The next part of fixtures with changed maintenance presets.

### CODE REVIEW

File: resources/fixtures/Stairville/Stairville-MH-100-Beam-36x3-LED.qxf [hunk 1]
```diff
@@ -19,60 +19,60 @@
  <Channel Name="White" Preset="IntensityWhite"/>
  <Channel Name="Strobe">
   <Group Byte="0">Shutter</Group>
-  <Capability Min="0" Max="9">Off</Capability>
-  <Capability Min="10" Max="140">Increasing speed</Capability>
-  <Capability Min="141" Max="255">Random speed</Capability>
+  <Capability Min="0" Max="9" Preset="ShutterClose">Off</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Stairville/Stairville-MH-100-Beam-36x3-LED.qxf:22]: I'm not sure this is correct. I believe the manual intends "strobe off", which in this case means "ShutterOpen"

File: resources/fixtures/Stairville/Stairville-MH-X50.qxf [hunk 2]
```diff
@@ -19,52 +19,52 @@
...
-  <Capability Res="ClayPaky/gobo00067.svg" Min="112" Max="119">Gobo 2 shake (speed)</Capability>
+  <Capability Min="0" Max="7" Preset="GoboMacro" Res1="Others/open.svg">Open</Capability>
+  <Capability Min="8" Max="15" Preset="GoboMacro" Res1="ClayPaky/gobo00067.svg">Gobo 2</Capability>
+  <Capability Min="16" Max="23" Preset="GoboMacro" Res1="ClayPaky/gobo00050.png">Gobo 3</Capability>
+  <Capability Min="24" Max="31" Preset="GoboMacro" Res1="Others/gobo00038.svg">Gobo 4</Capability>
+  <Capability Min="32" Max="39" Preset="GoboMacro" Res1="Others/gobo00039.svg">Gobo 5</Capability>
+  <Capability Min="40" Max="47" Preset="GoboMacro" Res1="ClayPaky/gobo00060.svg">Gobo 6</Capability>
+  <Capability Min="48" Max="55" Preset="GoboMacro" Res1="Others/gobo00040.svg">Gobo 7</Capability>
+  <Capability Min="56" Max="63" Preset="GoboMacro" Res1="Others/gobo00041.svg">Gobo 8</Capability>
+  <Capability Min="64" Max="71" Preset="GoboMacro" Res1="Others/gobo00041.svg">Gobo 8 shake (speed)</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Stairville/Stairville-MH-X50.qxf:61]: For these there is "GoboShakeMacro"

File: resources/fixtures/Stairville/Stairville-MH-X60th-LED-Spot.qxf [hunk 3]
```diff
@@ -31,55 +31,55 @@
...
-  <Capability Min="120" Max="127" Res="Others/open.svg">Open</Capability>
+  <Capability Min="0" Max="7" Preset="GoboMacro" Res1="Others/open.svg">Open</Capability>
+  <Capability Min="8" Max="15" Preset="GoboMacro" Res1="ClayPaky/gobo00067.svg">Gobo 2</Capability>
+  <Capability Min="16" Max="23" Preset="GoboMacro" Res1="Chauvet/gobo00015.png">Gobo 3</Capability>
+  <Capability Min="24" Max="31" Preset="GoboMacro" Res1="Chauvet/gobo00001.svg">Gobo 4</Capability>
+  <Capability Min="32" Max="39" Preset="GoboMacro" Res1="Chauvet/gobo00016.svg">Gobo 5</Capability>
+  <Capability Min="40" Max="47" Preset="GoboMacro" Res1="Chauvet/gobo00017.svg">Gobo 6</Capability>
+  <Capability Min="48" Max="55" Preset="GoboMacro" Res1="Chauvet/gobo00018.svg">Gobo 7</Capability>
+  <Capability Min="56" Max="63" Preset="GoboMacro" Res1="Chauvet/gobo00019.svg">Gobo 8</Capability>
+  <Capability Min="64" Max="71" Preset="GoboMacro" Res1="Chauvet/gobo00019.svg">Gobo 8 shake, increasing speed</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Stairville/Stairville-MH-X60th-LED-Spot.qxf:75]: GoboShakeMacro

File: resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf [hunk 4]
```diff
@@ -13,15 +13,15 @@
  <Channel Name="Tilt" Preset="PositionTilt"/>
  <Channel Name="Shutter">
   <Group Byte="0">Shutter</Group>
-  <Capability Min="0" Max="15">Open</Capability>
-  <Capability Min="16" Max="131">Strobe Increasing Speed</Capability>
-  <Capability Min="132" Max="139">Open</Capability>
+  <Capability Min="0" Max="15" Preset="ShutterOpen">Open</Capability>
+  <Capability Min="16" Max="131" Preset="StrobeSlowToFast">Strobe Increasing Speed</Capability>
+  <Capability Min="132" Max="139" Preset="ShutterOpen">Open</Capability>
   <Capability Min="140" Max="181">Slow Open to Fast Close</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf:19]: RampUpSlowToFast

File: resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf [hunk 5]
```diff
@@ -13,15 +13,15 @@
...
-  <Capability Min="182" Max="189">Open</Capability>
+  <Capability Min="182" Max="189" Preset="ShutterOpen">Open</Capability>
   <Capability Min="190" Max="231">Slow Close to Fast Open</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf:21]: RampDownSlowToFast

File: resources/fixtures/Stairville/Stairville-MH-X50.qxf (hunk 2 above)
Comment (kluszon) [reply resources/fixtures/Stairville/Stairville-MH-X50.qxf]: Fixed

File: resources/fixtures/Stairville/Stairville-MH-X60th-LED-Spot.qxf (hunk 3 above)
Comment (kluszon) [reply resources/fixtures/Stairville/Stairville-MH-X60th-LED-Spot.qxf]: Fixed

File: resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf (hunk 4 above)
Comment (kluszon) [reply resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf]: Fixed

File: resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf (hunk 5 above)
Comment (kluszon) [reply resources/fixtures/Stairville/Stairville-novaWash-Quad-LED.qxf]: Fixed

File: resources/fixtures/Stairville/Stairville-MH-100-Beam-36x3-LED.qxf (hunk 1 above)
Comment (kluszon) [reply resources/fixtures/Stairville/Stairville-MH-100-Beam-36x3-LED.qxf:22]: Agreed and fixed.

**Pull request merged**